content/for-administrators/how-to-guides/certificate-management/tls-certs.md

```diff
@@ -0,0 +1,162 @@
+# Configuring TLS Certificates Using Infra GitOps
+
+This document provides a step-by-step guide to configure TLS certificates for different tenants using Infra GitOps.
+
+## Step 1: Navigate to the Target Path
+
+Navigate to the appropriate path in your Infra GitOps repository. For this example, the path is:
+
+```plaintext
+<environment>/tenant-operator-config/templates/
```

Comment:
@owais-rehman this should be called cluster then and not environment; as environment is dev, stage and prod etc.